src/main/scala/beam/sim/config/BeamConfig.scala

```diff
@@ -499,6 +501,13 @@ object BeamConfig {
                     else 0.0,
                   transfer = if (c.hasPathOrNull("transfer")) c.getDouble("transfer") else -1.4,
                   transit_crowding = if (c.hasPathOrNull("transit_crowding")) c.getDouble("transit_crowding") else 0.0,
+                  transit_crowding_VOT_cutoff =
+                    if (c.hasPathOrNull("transit_crowding_VOT_cutoff")) c.getDouble("transit_crowding_VOT_cutoff")
+                    else 0.5,
+                  transit_crowding_VOT_multiplier =
+                    if (c.hasPathOrNull("transit_crowding_VOT_multiplier"))
+                      c.getDouble("transit_crowding_VOT_multiplier")
+                    else 1.0,
```

Comment:
@zneedell
am I right, that with this 1.0 default value of "transit_crowding_VOT_multiplier"  from now on we will always slightly consider crowding in any BEAM run?